woocommerce/woocommerce-android
# EU-Bound Shipping Notice: Add Tracks

Summary
==========
Fix issue #8981 by introducing three track events to the EU Shipping Notice flow:

- `EU_SHIPPING_NOTICE_SHOWN` when the Notice is presented to the user inside the `Create Shipping Label` view
- `EU_SHIPPING_NOTICE_DISMISSED` when the user dismiss the banner
- `EU_SHIPPING_NOTICE_LEARN_MORE_TAPPED` when the user taps the `Learn More` button

How to Test
==========
1. Open the app using a store with the Shipping Labels plugin activated
2. Open the Order Details of an order with the Processing status
3. Hit the Create Shipping Label button
4. Verify that the Banner only appears when the Origin country is US and the Destination country is one of the EU countries listed above
5. Verify that the `EU_SHIPPING_NOTICE_SHOWN` event is tracked
6. Hit the `Learn More` button and verify that the `EU_SHIPPING_NOTICE_LEARN_MORE_TAPPED` event is tracked
7. Dismiss the banner and verify that the `EU_SHIPPING_NOTICE_DISMISSED` event is tracked

Update release notes:
